Ansible/play_file

```diff
@@ -0,0 +1,91 @@
+---
+- name: Install To Do App on new webserver
+  hosts: managed_hosts
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: api_key
+      prompt: What is the api key?
+      private: no
+
+    - name: api_token
+      prompt: What is the api token?
+      private: no
+
+    - name: trellobid
+      prompt: What is the trellobid?
+      private: no    
+
+  tasks: 
+  - name: Install git on VM
+    ansible.builtin.yum:
+      name: git
+      state: present
+    become: yes
+
+  - name: Install Python on VM
+    ansible.builtin.yum:
+      name: python
+      state: latest
+    become: yes
+
+  - name: Check Poerty on VM
+    ansible.builtin.shell:
+      cmd: PATH=$PATH:/etc/poetry/bin; which poetry
+    register: poetry_exists
+    failed_when: no
```

Comment:
I believe functionally the same, but I feel like the [`ignore-errors`](https://docs.ansible.com/ansible/latest/user_guide/playbooks_error_handling.html#ignoring-failed-commands) argument is more intuitive than `failed_when: no`